<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNewWindow"/>
    <addaction name="actionNewTab"/>
    <addaction name="actionOpenUrl"/>
    <addaction name="actionOpenFile"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionSendLink"/>
    <addaction name="separator"/>
    <addaction name="actionPrintPreview"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionSelectAll"/>
    <addaction name="actionSearch"/>
    <addaction name="actionSearchNext"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuZoom">
     <property name="title">
      <string>Zoom</string>
     </property>
     <addaction name="actionZoomIn"/>
     <addaction name="actionZoomOut"/>
     <addaction name="actionZoomNormal"/>
     <addaction name="separator"/>
     <addaction name="actionTextOnly"/>
    </widget>
    <addaction name="actionStop"/>
    <addaction name="actionReload"/>
    <addaction name="separator"/>
    <addaction name="menuZoom"/>
    <addaction name="actionSource"/>
    <addaction name="actionConsole"/>
    <addaction name="actionDownloads"/>
    <addaction name="actionCookies"/>
    <addaction name="separator"/>
    <addaction name="actionFullScreen"/>
   </widget>
   <widget class="QMenu" name="menu_History">
    <property name="title">
     <string>&amp;History</string>
    </property>
    <addaction name="actionBack"/>
    <addaction name="actionForward"/>
    <addaction name="actionHome"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_Bookmarks">
    <property name="title">
     <string>&amp;Bookmarks</string>
    </property>
    <addaction name="actionAddBookmark"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_View"/>
   <addaction name="menu_History"/>
   <addaction name="menu_Bookmarks"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="navBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionBack"/>
   <addaction name="actionForward"/>
   <addaction name="actionReload"/>
   <addaction name="actionStop"/>
   <addaction name="actionHome"/>
  </widget>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="../../libsrc/phia/phia.qrc">
     <normaloff>:/gnome/undo</normaloff>:/gnome/undo</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="../../libsrc/phia/phia.qrc">
     <normaloff>:/gnome/redo</normaloff>:/gnome/redo</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="../../libsrc/phia/phia.qrc">
     <normaloff>:/gnome/cut</normaloff>:/gnome/cut</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="../../libsrc/phia/phia.qrc">
     <normaloff>:/gnome/copy</normaloff>:/gnome/copy</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="../../libsrc/phia/phia.qrc">
     <normaloff>:/gnome/paste</normaloff>:/gnome/paste</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="../../libsrc/phia/phia.qrc">
     <normaloff>:/gnome/error</normaloff>:/gnome/error</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Select all</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../libsrc/phia/phia.qrc">
     <normaloff>:/gnome/search</normaloff>:/gnome/search</iconset>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionSearchNext">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../libsrc/phia/phia.qrc">
     <normaloff>:/gnome/forward</normaloff>:/gnome/forward</iconset>
   </property>
   <property name="text">
    <string>Search next</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="../../libsrc/phia/phia.qrc">
     <normaloff>:/gnome/config</normaloff>:/gnome/config</iconset>
   </property>
   <property name="text">
    <string>User profile</string>
   </property>
  </action>
  <action name="actionNewWindow">
   <property name="icon">
    <iconset resource="../../libsrc/phia/phia.qrc">
     <normaloff>:/gnome/newwindow</normaloff>:/gnome/newwindow</iconset>
   </property>
   <property name="text">
    <string>New window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionNewTab">
   <property name="icon">
    <iconset resource="../../libsrc/phia/phia.qrc">
     <normaloff>:/gnome/addfolder</normaloff>:/gnome/addfolder</iconset>
   </property>
   <property name="text">
    <string>New tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionOpenUrl">
   <property name="icon">
    <iconset resource="../../libsrc/phia/phia.qrc">
     <normaloff>:/gnome/world</normaloff>:/gnome/world</iconset>
   </property>
   <property name="text">
    <string>Open url</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="icon">
    <iconset resource="../../libsrc/phia/phia.qrc">
     <normaloff>:/gnome/folder</normaloff>:/gnome/folder</iconset>
   </property>
   <property name="text">
    <string>Open file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="../../libsrc/phia/phia.qrc">
     <normaloff>:/gnome/cancel</normaloff>:/gnome/cancel</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="../../libsrc/phia/phia.qrc">
     <normaloff>:/gnome/saveas</normaloff>:/gnome/saveas</iconset>
   </property>
   <property name="text">
    <string>Save page as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSendLink">
   <property name="icon">
    <iconset resource="../../libsrc/phia/phia.qrc">
     <normaloff>:/gnome/mailforward</normaloff>:/gnome/mailforward</iconset>
   </property>
   <property name="text">
    <string>Send link</string>
   </property>
  </action>
  <action name="actionPrintPreview">
   <property name="icon">
    <iconset resource="../../libsrc/phia/phia.qrc">
     <normaloff>:/gnome/printpreview</normaloff>:/gnome/printpreview</iconset>
   </property>
   <property name="text">
    <string>Print preview</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="../../libsrc/phia/phia.qrc">
     <normaloff>:/gnome/print</normaloff>:/gnome/print</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../../libsrc/phia/phia.qrc">
     <normaloff>:/gnome/quit</normaloff>:/gnome/quit</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="../../libsrc/phia/phia.qrc">
     <normaloff>:/gnome/warning</normaloff>:/gnome/warning</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="icon">
    <iconset resource="../../libsrc/phia/phia.qrc">
     <normaloff>:/gnome/reload</normaloff>:/gnome/reload</iconset>
   </property>
   <property name="text">
    <string>Reload</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="../../libsrc/phia/phia.qrc">
     <normaloff>:/gnome/zoomout</normaloff>:/gnome/zoomout</iconset>
   </property>
   <property name="text">
    <string>Zoom in</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="../../libsrc/phia/phia.qrc">
     <normaloff>:/gnome/zoomin</normaloff>:/gnome/zoomin</iconset>
   </property>
   <property name="text">
    <string>Zoom out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionZoomNormal">
   <property name="icon">
    <iconset resource="../../libsrc/phia/phia.qrc">
     <normaloff>:/gnome/zoomnormal</normaloff>:/gnome/zoomnormal</iconset>
   </property>
   <property name="text">
    <string>Zoom normal</string>
   </property>
  </action>
  <action name="actionTextOnly">
   <property name="icon">
    <iconset resource="../../libsrc/phia/phia.qrc">
     <normaloff>:/gnome/textonly</normaloff>:/gnome/textonly</iconset>
   </property>
   <property name="text">
    <string>Text only</string>
   </property>
  </action>
  <action name="actionSource">
   <property name="icon">
    <iconset resource="../../libsrc/phia/phia.qrc">
     <normaloff>:/gnome/html</normaloff>:/gnome/html</iconset>
   </property>
   <property name="text">
    <string>Page source</string>
   </property>
  </action>
  <action name="actionFullScreen">
   <property name="icon">
    <iconset resource="../../libsrc/phia/phia.qrc">
     <normaloff>:/gnome/fullscreen</normaloff>:/gnome/fullscreen</iconset>
   </property>
   <property name="text">
    <string>Full screen</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="icon">
    <iconset resource="../../libsrc/phia/phia.qrc">
     <normaloff>:/gnome/left</normaloff>:/gnome/left</iconset>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
   <property name="shortcut">
    <string>Alt+Left</string>
   </property>
  </action>
  <action name="actionForward">
   <property name="icon">
    <iconset resource="../../libsrc/phia/phia.qrc">
     <normaloff>:/gnome/right</normaloff>:/gnome/right</iconset>
   </property>
   <property name="text">
    <string>Forward</string>
   </property>
   <property name="shortcut">
    <string>Alt+Right</string>
   </property>
  </action>
  <action name="actionHome">
   <property name="icon">
    <iconset resource="../../libsrc/phia/phia.qrc">
     <normaloff>:/gnome/home</normaloff>:/gnome/home</iconset>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="../../libsrc/phia/phia.qrc">
     <normaloff>:/gnome/help</normaloff>:/gnome/help</iconset>
   </property>
   <property name="text">
    <string>Online Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../../libsrc/phia/phia.qrc">
     <normaloff>:/gnome/info</normaloff>:/gnome/info</iconset>
   </property>
   <property name="text">
    <string>About Amphibia</string>
   </property>
  </action>
  <action name="actionCookies">
   <property name="icon">
    <iconset resource="../../libsrc/phia/phia.qrc">
     <normaloff>:/gnome/cookie</normaloff>:/gnome/cookie</iconset>
   </property>
   <property name="text">
    <string>Cookies</string>
   </property>
  </action>
  <action name="actionAddBookmark">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../libsrc/phia/phia.qrc">
     <normaloff>:/gnome/bookmark</normaloff>:/gnome/bookmark</iconset>
   </property>
   <property name="text">
    <string>Add bookmark</string>
   </property>
  </action>
  <action name="actionDownloads">
   <property name="icon">
    <iconset resource="../../libsrc/phia/phia.qrc">
     <normaloff>:/gnome/inetfolder</normaloff>:/gnome/inetfolder</iconset>
   </property>
   <property name="text">
    <string>Downloads</string>
   </property>
  </action>
  <action name="actionConsole">
   <property name="icon">
    <iconset resource="../../libsrc/phia/phia.qrc">
     <normaloff>:/gnome/script</normaloff>:/gnome/script</iconset>
   </property>
   <property name="text">
    <string>JavaScript console</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../libsrc/phia/phia.qrc"/>
 </resources>
 <connections/>
</ui>
